catalogue_pipeline/id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/services/IdMinterWorkerService.scala

```diff
@@ -28,8 +28,10 @@ class IdMinterWorkerService @Inject()(
     for {
       json <- Future.fromTry(parseMessageIntoJson(message))
       workWithCanonicalId <- idEmbedder.embedId(json)
-      _ <- writer.writeMessage(workWithCanonicalId.toString(),
-                               Some(snsSubject))
+      _ <- writer.writeMessage(
+        message = workWithCanonicalId.toString(),
+        subject = "source: IdMinterWorkerService.processMessage"
```

Comment:
I'd rather not hardcode the name of the service in a string. It's easy to forget renaming it if we refactor. You could use `this.getClass.getSimpleName` instead

Reply:
I didn’t realise you could do that, thanks!